client.cpp

```diff
@@ -1,37 +1,77 @@
-#include <hazelcast/client/hazelcast_client.h>
 #include <string>
+#include "model/City.h"
+#include "model/Country.h"
 
+/**
+ * This is boilerplate application that configures client to connect Hazelcast
+ * Cloud cluster.
+ * <p>
+ * See: <a href="https://docs.hazelcast.com/cloud/cpp-client">https://docs.hazelcast.com/cloud/cpp-client</a>
+ */
+
+void mapExample(hazelcast::client::hazelcast_client client);
```

Comment:
Forgive my lack of C++ knowledge :smile: Are we using snake case in the parameters and camel case in the function names? Or should all of it be in snake case?

Reply:
done